build.sbt

```diff
@@ -55,4 +55,7 @@ lazy val sparkCommonsTest = (projectMatrix in file("spark-commons-test"))
   )
   .jvmPlatform(scalaVersions = Seq(scala211, scala212))
 
-releasePublishArtifactsAction := PgpKeys.publishSigned.value
+// licenceHeader check:
+ThisBuild / organizationName := "ABSA Group Limited"
+ThisBuild / startYear := Some(2021)
+ThisBuild / licenses += "Apache-2.0" -> url("https://www.apache.org/licenses/LICENSE-2.0.txt")
```

Comment:
If I understood it correctly, these three values should be used also for https://github.com/sbt/sbt-header#getting-started, so it might make sense to group them together. So the question is whether we want to put them into `build.sbt` or `publish.sbt`. 

Since this header check is closer to the compilation/validation part, not the publishing part, I think that `build.sbt` is a better candidate - additionally, it would be consistent with our spark-data-standardization: https://github.com/AbsaOSS/spark-data-standardization/blob/master/build.sbt#L63-L66

(btw, our https://github.com/AbsaOSS/sbt-git-hooks/tree/master/.github/workflows doesn't have a licence check - should we add it there & use similar logic?)

Of course, as always, I'm happy to be persuaded, I don't insist on this.

Reply:
Changed, based on our discussion elsewhere.